language/stdlib/modules/LibraAccount.move

```diff
@@ -883,18 +883,20 @@ module LibraAccount {
         creator_account: &signer,
         new_account_address: address,
         auth_key_prefix: vector<u8>,
+        human_name: vector<u8>,
     ) {
         assert(Roles::has_libra_root_role(creator_account), ENOT_LIBRA_ROOT);
         let new_account = create_signer(new_account_address);
         Event::publish_generator(&new_account);
-        ValidatorConfig::publish(&new_account, creator_account);
+        ValidatorConfig::publish(&new_account, creator_account, human_name);
         make_account(new_account, auth_key_prefix)
     }
 
     public fun create_validator_operator_account(
         creator_account: &signer,
         new_account_address: address,
         auth_key_prefix: vector<u8>,
+        _human_name: vector<u8>,
```

Comment:
I am not sure what is the best way to add names: we can add another module `ValidatorOperatorConfig` with a human name in it, but I think a better solution is to add a universal module `NamedAccount` and make both validator accounts and validator operators accounts instantiate it. @sblackshear, what do you think?

Reply:
Hmm, good question. I do like `NamedAccount`. The problem is that we already have `DualAttestation::Credential`, which has human-readable names for VASPs/DDs. We could take the names out of there and use `NamedAccount` instead (and I think that would be better), but we can't because that would be a client-breaking change...

I think given that constraint, the most consistent way is probably to have the names live in the role-specific config? Maybe `ValidatorOperatorConfig` could be part of `ValidatorConfig` instead of a new module, and we could add some other useful info to it. E.g., we could add the number of validators that each operator is currently responsible for; that would let us do some on-chain enforcement of the "an operator shouldn't run > 1/3 nodes" rule.